apps/comps/wagmi-config.ts

```diff
@@ -1,4 +1,9 @@
 import { getDefaultConfig } from "@rainbow-me/rainbowkit";
+import {
+  coinbaseWallet,
+  okxWallet,
+  walletConnectWallet,
```

Comment:
i just noticed this behavior: if i select the WalletConnect option, exit w/o connecting, and refresh the page...we see some logged errors
<img width="1512" alt="Screenshot 2025-07-02 at 1 27 16 PM" src="https://github.com/user-attachments/assets/b3c7f12b-c8d3-4fa9-adda-e75d9cd3c321" />

Reply:
it's only for WalletConnect, and i've seen this issue happen elsewhere. it's something we can't control, though, so we can probably keep it...unless you know if a quick fix